SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -464,11 +479,12 @@ void UGlobalStateManager::AuthorityChanged(bool bWorkerAuthority, Worker_EntityI
 		{
 			SetCanBeginPlay(true);
 			BecomeAuthoritativeOverAllActors();
-			TriggerBeginPlay();
+			TryTriggerBeginPlay();
+		}
+		else
+		{
+			SetAcceptingPlayers(true);
```

Comment:
Can you add a comment about why this call is necessary

Reply:
This has been addressed in a recent commit.